pom.xml

```diff
@@ -114,6 +118,8 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>scm-api</artifactId>
+      <!-- TODO: Replace with release of https://github.com/jenkinsci/scm-api-plugin/pull/282 once it is available -->
```

Comment:
https://github.com/jenkinsci/scm-api-plugin/releases/tag/698.v8e3b_c788f0a_6?

Reply:
87fcaa7390e5a83ce6dd6ee50d3f0db49210e2e2 includes that.  Friday's release of plugin BOM will allow that version line to be removed.